lighthouse-core/audits/estimated-input-latency.js

```diff
@@ -50,10 +52,14 @@ class EstimatedInputLatency extends Audit {
       // Use speedline's first paint as start of range for input latency check.
       const startTime = artifacts.Speedline.first;
 
-      const trace = artifacts.traceContents;
+      const traceContents = artifacts[TRACE_NAME] && artifacts[TRACE_NAME].traceContents;
+      if (!traceContents || !Array.isArray(traceContents)) {
```

Comment:
No need for this check. `tracingProcessor.init()` already throws on non-Array input